examples/table/sortable-table.html

```diff
@@ -0,0 +1,227 @@
+      The example uses HTML table markup for all elements of the table structure, e.g., cells, rows, column headers, and caption.
+      The <code>aria-sort</code> attribute is set on the column header of the currently sorted column, and the header text of sortable columns is wrapped in a <code>button</code> element.
+      One column, the <q>Address</q> column is not sortable.
+    </p>
+    <p>Similar examples include: </p>
+    <ul>
+      <li><a href="table.html">Table Example</a>: A table constructed using ARIA roles to convey  table semantics.</li>
+      <li><a href="../grid/dataGrids.html">Data Grid Examples</a>: Three example implementations of grid that include features relevant to presenting tabular information, such as content editing, sort, and column hiding.</li>
+    </ul>
+
+    <section>
+      <h2>Example Option</h2>
+      <label>
+        <input type="checkbox"
+          value="show-unsorted-icon"
+          aria-describedby="show-unsorted-icon-desc">
+        Display sort icon in headers of columns that are not sorted
+      </label>
+      <p id="show-unsorted-icon-desc">
+        Adds an <q>♢</q> icon to the header of each column that is sortable but is not currently sorted.
+        Some implementations do this to help users distinguish sortable columns from columns that cannot be sorted.
+        It is important that a different shaped icon is used for users with visual impairments to perceive the difference with the sorted column icon in high contrast modes.
+      </p>
+    </section>
+
+
+    <section>
+      <div class="example-header">
+        <h2 id="ex_label">Example</h2>
+      </div>
+      <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
+      <div id="ex1">
+        <table class="sortable">
+          <caption>
+            Students currently enrolled in WAI-ARIA 101
+            <span class="sr-only">, column headers with buttons are sortable.</span>
+          </caption>
+          <thead>
+            <tr>
+              <th>
+                <button>
+                  First Name
+                  <span aria-hidden="true"></span>
+                </button>
+              </th>
+              <th aria-sort="ascending">
+                <button>
+                  Last Name
+                  <span aria-hidden="true"></span>
+                </button>
+              </th>
+              <th>
+                <button>
+                  Company
+                  <span aria-hidden="true"></span>
+                </button>
+              </th>
+              <th class="no-sort">Address</th>
+              <th class="num">
+                <button>
+                  Favorite Number
+                  <span aria-hidden="true"></span>
+                </button>
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td role="cell">Fred</td>
```

Comment:
Why declare the role? It is implicit. The spec says not to do this. Pretty certain the role declarations should be removed.